Assets/__Scripts/MapEditor/Mapping/PlacementControllers/PlacementController.cs

```diff
@@ -215,13 +215,6 @@ private void FinishDrag()
 
     protected virtual void Update()
     {
-        if ((isDraggingObject && !Input.GetMouseButton(0)) || (isDraggingObjectAtTime && !Input.GetMouseButton(1)))
```

Comment:
This is not duplicate code, it handles an edge case where the buttons are released outside the window

Reply:
Ah figured it was left behind from the old input system, will have to add a check for that then since it was causing 2 modified actions to be added for the same movement.

Reply:
Do let me know when you find a way to incorporate both.

Reply:
Fixed it turns out the duplicate drag was because both the note placement controller and bomb placement controller were dragging the same note. Fixed by checking the objects type.